algviz/docs/list/listVisualization.svg

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Nitpick:  Convention in python is to use underscores instead of camel-case for filenames, and I'd prefer for the non-python files to follow the same convention.

Reply:
Yup, I can do that